examples/manifold/plot_manifold_sphere.py

```diff
@@ -140,11 +137,15 @@
 ax.yaxis.set_major_formatter(NullFormatter())
 plt.axis("tight")
 
+import warnings  # TODO(1.2) Remove.
+
 # Perform t-distributed stochastic neighbor embedding.
-t0 = time()
-tsne = manifold.TSNE(n_components=2, init="pca", random_state=0)
-trans_data = tsne.fit_transform(sphere_data).T
-t1 = time()
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore", FutureWarning)
+    t0 = time()
+    tsne = manifold.TSNE(n_components=2, init="pca", random_state=0)
```

Comment:
This was actually my original approach to handle the warnings, but based on the final paragraph of [the deprecation subsection](https://scikit-learn.org/stable/developers/contributing.html#deprecation) of the contribution guidelines, I reasoned that we would always want to handle all FutureWarning cases to prevent them from showing in the examples. As this was a more concise way to cover both warnings, I opted for this more general strategy.

With that being said, I also tend to favor specificity in most cases and have made the changes you suggested.